src/check-for-spam.js

```diff
@@ -0,0 +1,41 @@
+// content refers to the contents of the blog post (article) sent by feedparser
+module.exports = function (content) {
+  // Minimum words set to 20, can be changed
+  const MINWORDS = 20;
```

Comment:
Excellent use of constants. I like this. I would suggest creating a file with all the constants definitions and then importing from there. This would be one of them. For now this is good though. If you'd like to work on another issue you can create one for this and self-assign

Reply:
I'm going to leave MINWORDS in the compenent file for now, as I feel like we don't need an entirely separate file for one constant.
That being said, if there are a lot more constants in other people's components, making a separate file would be a good idea.